SwiftVoiceCallKitQuickstart/ViewController.swift

```diff
@@ -444,27 +444,12 @@ class ViewController: UIViewController, PKPushRegistryDelegate, TVONotificationD
             return
         }
         
-        call = TwilioVoice.call(accessToken, params: [:], delegate: self)
-        
-        guard let call = call else {
-            NSLog("Failed to start outgoing call")
-            completionHandler(false)
-            return
-        }
-        
-        call.uuid = uuid
-        
+        call = TwilioVoice.call(accessToken, params: [:], uuid:uuid, delegate: self)
         self.callKitCompletionCallback = completionHandler
     }
     
     func performAnswerVoiceCall(uuid: UUID, completionHandler: @escaping (Bool) -> Swift.Void) {
-        guard let call = self.callInvite?.accept(with: self) else {
-            completionHandler(false)
-            return
-        }
-        
-        call.uuid = uuid
-        
+        call = self.callInvite?.accept(with: self)
```

Comment:
Same as Obj-C PR, should we do anything with the UUID argument like assert that it is the same as the callInvite?